scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixEvaluation.java

```diff
@@ -0,0 +1,32 @@
+package scalafix.interfaces;
+
+import java.util.Optional;
+
+public interface ScalafixEvaluation {
+    /**
+     * @return boolean true if scalafix has successfully been evaluated on all files configured.
+     */
+    boolean isSuccessful();
+
+    /**
+     * @return List of scalafixErrors resulted from scalafix evaluation
+     */
+    ScalafixError[] getErrors();
+
+    /**
+     * @return A message error is provided if the run is not successful
+     */
+    Optional<String> getMessageError();
+
+    /**
+     * @return ScalafixOutput[] for each file we store the scalafix evaluation: If the evaluation is successful,
```

Comment:
is the type really needed in javadoc `@return` comments?
```suggestion
     * @return for each file we store the scalafix evaluation: If the evaluation is successful,
```

Reply:
no, type is not necessary in `return` java doc